recipes/bamtools/meta.yaml

```diff
@@ -1,15 +1,19 @@
 {% set version = "2.5.1" %}
-{% set sha256 = "4abd76cbe1ca89d51abc26bf43a92359e5677f34a8258b901a01f38c897873fc" %}
+
 package:
   name: bamtools
   version: '{{version}}'
 
 source:
-  url: https://github.com/pezmaster31/bamtools/archive/v{{version}}.tar.gz
-  sha256: '{{sha256}}'
+  #url: https://github.com/pezmaster31/bamtools/archive/v{{version}}.tar.gz
+  # This is no longer maintained. The most recent commits (from a year ago) seem to fix support for modern C++
+  url: https://github.com/pezmaster31/bamtools/archive/2391b1a1275816ad89c624586fa02b1a621924f5.zip
+  sha256: 6c1f6fcae6584597700ac8429c1fcdfac2436460b394eb9cbc2fdf813d3957e2
```

Comment:
This apparently broke downstream packages, see gh-28020.
I suppose we likely should pull the `bamtools=2.5.1[build_number=8]` builds (may need to pull built downstream packages that linked to this too then, I guess :/) and either figure out how to fix the upstream `2.5.1` build or change the version of this to `2.5.2.dev0` or similar.

Reply:
crap yeah making this 2.5.2.dev0 seems reasonable. The problem is that we then have to patch the 2.5.1 source so it's buildable. I really wish people would stop using bamtools...

Reply:
Looks like it compiled with only pulling one upstream patch: https://github.com/bioconda/bioconda-recipes/pull/28036

Shall we just merge that one and move the build 8 to `broken`? Any idea how many downstream packages that link to libbamtools have been built already?

Reply:
Yeah, let's move 8 to broken. I'm not sure how many packages actually dynamically link to bamtools, most seem to use the .a file.